src/docs/sdk/features.mdx

```diff
@@ -203,6 +203,38 @@ If the SDK is being [rate-limited](/sdk/rate-limiting/), which causes the SDK to
 - [Objective-C](https://github.com/getsentry/sentry-cocoa/blob/master/Sources/Sentry/SentryHttpTransport.m)
 - [TypeScript](https://github.com/getsentry/sentry-electron/blob/master/src/main/transports/electron-offline-net.ts)
 
+## Start-Up Crash Detection
+
+We recommend implementing this feature for mobile SDKs.
+
+If the application crashes shortly after the init of the SDK, the SDK should provide a mechanism to guarantee transmission
+to Sentry. Ideally, SDKs could send the events in a separate process not impacted by the crashing application. With the
+limitations on mobile platforms, spawning an extra process only for sending envelopes is hard to achieve or impossible.
+The SDKs on these platforms send envelopes on a background thread to not block the UI thread. A crash occurring shortly
```

Comment:
> on a background thread to not block the UI thread

Not only because of that, on Android, you can't do network calls in the main thread, maybe add a note about that.